Loading addresses fails on CI

After merging a fix for the downloading part, I have merged the PR into main and encountered this error when retriving the addresses https://github.com/oceanprotocol/ocean.py/actions/runs/3523080688/jobs/5906809502#step:7:7717

```
@enforce_types
    def get_contracts_addresses(config: dict) -> Optional[Dict[str, str]]:
        """Get addresses for all contract names, per network and address_file given."""
        network_name = config["NETWORK_NAME"]
        addresses = get_addresses_with_fallback(config)
    
        network_addresses = [val for key, val in addresses.items() if key == network_name]
    
        if network_addresses is None:
            raise Exception(
                f"Address not found for {network_name}. Please check your address file."
            )
    
>       return _checksum_contract_addresses(network_addresses=network_addresses[0])
E       IndexError: list index out of range

ocean_lib/web3_internal/contract_utils.py:72: IndexError
```

--- a/ocean_lib/web3_internal/contract_utils.py
+++ b/ocean_lib/web3_internal/contract_utils.py
@@ -64,7 +64,7 @@ def get_contracts_addresses(config: dict) -> Optional[Dict[str, str]]:
 
     network_addresses = [val for key, val in addresses.items() if key == network_name]
 
-    if network_addresses is None:
+    if network_addresses is None or len(network_addresses) == 0:
         raise Exception(
             f"Address not found for {network_name}. Please check your address file."
         )

--- a/ocean_lib/web3_internal/contract_utils.py
+++ b/ocean_lib/web3_internal/contract_utils.py
@@ -64,7 +64,7 @@ def get_contracts_addresses(config: dict) -> Optional[Dict[str, str]]:
 
     network_addresses = [val for key, val in addresses.items() if key == network_name]
 
-    if network_addresses is None or len(network_addresses) == 0:
+    if not network_addresses:
         raise Exception(
             f"Address not found for {network_name}. Please check your address file."
         )

